usersimcrs/user_modeling/preference_model.py

```diff
@@ -40,7 +56,6 @@ def __init__(
         domain: Domain,
         item_collection: ItemCollection,
         historical_ratings: Ratings,
-        model_variant: PreferenceModelVariant,
```

Comment:
Please update the initilization of `PreferenceModel` in `run_simulation.py` and remove `model_variant` from docstring.